File: src/com/classe/voyage/Employe.java
package com.classe.voyage;

import java.lang.Integer;
import java.lang.Object;
import java.lang.String;

public class Employe {
  private Integer id;

  private String nom;

  private String prenom;

  private Object id_genre;

  private Object date_naissance;

  private Object date_embauche;

  private Object id_poste;

  public Employe(Integer id, String nom, String prenom, Object id_genre, Object date_naissance,
      Object date_embauche, Object id_poste) {
    this.id = id;
    this.nom = nom;
    this.prenom = prenom;
    this.id_genre = id_genre;
    this.date_naissance = date_naissance;
    this.date_embauche = date_embauche;
    this.id_poste = id_poste;
  }

  public Integer getId() {
    return id;
  }

  public void setId(Integer id) {
    this.id = id;
  }

  public String getNom() {
    return nom;
  }

  public void setNom(String nom) {
    this.nom = nom;
  }

  public String getPrenom() {
    return prenom;
  }

  public void setPrenom(String prenom) {
    this.prenom = prenom;
  }

  public Object getId_genre() {
    return id_genre;
  }

  public void setId_genre(Object id_genre) {
    this.id_genre = id_genre;
  }

  public Object getDate_naissance() {
    return date_naissance;
  }

  public void setDate_naissance(Object date_naissance) {
    this.date_naissance = date_naissance;
  }

  public Object getDate_embauche() {
    return date_embauche;
  }

  public void setDate_embauche(Object date_embauche) {
    this.date_embauche = date_embauche;
  }

  public Object getId_poste() {
    return id_poste;
  }

  public void setId_poste(Object id_poste) {
    this.id_poste = id_poste;
  }
}

File: src/com/cs/voyage/Employe.cs
public class Employe {
 {
    private int id;
    private string nom;
    private string prenom;
    private object id_genre;
    private object date_naissance;
    private object date_embauche;
    private object id_poste;

    public Employe(int id, string nom, string prenom, object id_genre, object date_naissance, object date_embauche, object id_poste) {
        this.id = id;
        this.nom = nom;
        this.prenom = prenom;
        this.id_genre = id_genre;
        this.date_naissance = date_naissance;
        this.date_embauche = date_embauche;
        this.id_poste = id_poste;
    }
    public int Id {
        get { return id; }
        set { id = value; }
    }
    public string Nom {
        get { return nom; }
        set { nom = value; }
    }
    public string Prenom {
        get { return prenom; }
        set { prenom = value; }
    }
    public object Id_genre {
        get { return id_genre; }
        set { id_genre = value; }
    }
    public object Date_naissance {
        get { return date_naissance; }
        set { date_naissance = value; }
    }
    public object Date_embauche {
        get { return date_embauche; }
        set { date_embauche = value; }
    }
    public object Id_poste {
        get { return id_poste; }
        set { id_poste = value; }
    }
}
    }
}